src/System.Security.Cryptography.X509Certificates/ref/project.json

```diff
@@ -3,9 +3,9 @@
     "System.IO": "4.0.0",
     "System.Runtime": "4.0.0",
     "System.Runtime.Handles": "4.0.0",
-    "System.Security.Cryptography.Primitives": "4.0.0-rc4-24215-01",
-    "System.Security.Cryptography.Algorithms": "4.2.0-rc4-24215-01",
-    "System.Security.Cryptography.Encoding": "4.0.0-rc4-24215-01"
+    "System.Security.Cryptography.Primitives": "4.0.1-beta-24217-04",
+    "System.Security.Cryptography.Algorithms": "4.2.1-beta-24217-04",
+    "System.Security.Cryptography.Encoding": "4.0.1-beta-24217-04"
```

Comment:
The src projects seem to be switching back to stable dependencies, and I suppose I can see an argument for the test projects requesting nearly-live dependencies; but I don't understand what position we seem to be taking with the ref dependencies